src/nft/utils/listNfts.ts

```diff
@@ -238,14 +244,15 @@ export async function signListing(
       }
       const order = createSellOrder(signerAddress, asset.expirationTime, [orderItem])
       try {
+        const prevOrderId = await getOrderId(asset.asset_contract.address, asset.tokenId)
         await signOrderData(provider, order)
         const payload: OrderPayload = {
           order: encodeOrder(order),
           isBundle: false,
           bundleName: '',
           bundleDesc: '',
-          orderIds: [],
-          changePrice: false,
+          orderIds: prevOrderId ? [prevOrderId] : [],
+          changePrice: Boolean(prevOrderId),
```

Comment:
non-blocking: instead of casting you could `!!`

Reply:
Boolean is actually faster as of 2020